### PR TITLE
Plugin-Admin

### PR DESCRIPTION
**Pull request by TilmanHaupt**
Plugin which shows selected data from the pluginConfig, and aims to allow to edit some of this data in the future.

### CODE REVIEW

File: ui/plugin-admin/src/components/Plugin.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  return (
+    <DataGridRow
+      key={plugin.id}
+      onClick={showDetails}
+      className={`cursor-pointer ${
+        showDetailsFor === plugin.id ? "active" : ""
+      } ${plugin?.disabled ? "text-theme-disabled" : ""} `}
+    >
+      <DataGridCell>
+        <p>{plugin.name}</p>
```
Comment (edda) [inline ui/plugin-admin/src/components/Plugin.js:33]: wrapping the cell's content in a `<p>` tag is not necessary. The only reason you would want to wrap a cell's content in a wrapper element is if you have multiple children within the cell and you don't want the default vertical flexbox layout applied to it. See documentation for the `cellVerticalAlignment` prop of the `DataGrid` component.

File: ui/plugin-admin/src/components/Plugin.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      </DataGridCell>
+      <DataGridCell>
+        {plugin.clusterName ? <p>{plugin.clusterName}</p> : <p>&mdash;</p>}
```
Comment (edda) [inline ui/plugin-admin/src/components/Plugin.js:36]: same as above with the wrapping `<p>`

File: ui/plugin-admin/src/components/Plugin.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+                className={`${
+                  plugin?.disabled ? "text-theme-link text-opacity-50" : ""
+                } `}
+              >
+                {url.name + " "}
+              </a>
+            )
+          })
+        ) : (
+          <p>&mdash;</p>
```
Comment (edda) [inline ui/plugin-admin/src/components/Plugin.js:56]: unnecessary `<p>` wrapper

File: ui/plugin-admin/src/components/Plugin.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        )}
+      </DataGridCell>
+      <DataGridCell>
+        <p>
```
Comment (edda) [inline ui/plugin-admin/src/components/Plugin.js:60]: `<p>` wrapper :)

File: ui/plugin-admin/src/components/PluginDetail.js [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+                  <DataGridCell>
+                    {plugin?.externalServicesUrls?.map((url) => {
+                      return (
+                        <a
+                          href={url.url}
+                          target="_blank"
+                          rel="noreferrer"
+                          key={url.url}
+                        >
+                          {url.name}{" "}
```
Comment (edda) [inline ui/plugin-admin/src/components/PluginDetail.js:101]: Not a fan of the `{" "}`. If you want to create margin to potential siblings, please use css. E.g.
```jsx
<a
  href=...
  ...
  className="mr-3"
>
```

File: ui/plugin-admin/src/components/PluginDetail.js [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+                        </a>
+                      )
+                    })}{" "}
```
Comment (edda) [inline ui/plugin-admin/src/components/PluginDetail.js:104]: As above, please no spacing via `{" "}`

**Review by edda — CHANGES_REQUESTED** [review]
Some small things :)

**Review by edda — APPROVED** [review]
+1

**Review by IvoGoman — APPROVED** [review]

**Pull request merged**